doc/releases.md

```diff
@@ -246,6 +246,8 @@ Use `tools/release.sh` to promote and sign the build. When run, it will perform
 
 If you didn't wait for ARM builds in the previous step before promoting the release, you should re-run `tools/release.sh` after the ARM builds have finished. That will move the ARM artifacts into the correct location. You will be prompted to re-sign SHASUMS256.txt.
 
+Note: if you forgot to push the tag before promoting the builds, you can tell the script to only sign a particular release by running `./tools/release.sh -s vX.Y.Z`.
```

Comment:
nit: may be reasons other than forgetting for signing after the fact. Perhaps it should just note the option and ability to sign after promoting?